### ISSUE TITLE
Bad start page appears when using "user_data_dir" with "uc" mode

### ISSUE THREAD

**Issue by M-Zubair10** [seleniumbase/SeleniumBase#1569]
using chrome data-dir give me this error for a sec and then continue onward

This doesn't save website cookies etc, for later auto sign-in

![image](https://user-images.githubusercontent.com/47709300/198332128-8721d8d8-dc32-42a0-be55-47d5a2a1a975.png)

I am using `Driver` function of seleniumbase

**Comment by mdmintz** [comment]
This is how you set the ``user_data_dir`` when using ``Driver()``:

```python
from seleniumbase import Driver

driver = Driver(browser="chrome",  user_data_dir=USER_DATA_DIR)
```

**Comment by M-Zubair10** [comment]
Yes exactly,
Like this
In fr"sitedata"

**Comment by mdmintz** [comment]
The correct call would be:

```python
from seleniumbase import Driver

driver = Driver(browser="chrome", uc=True, user_data_dir=USER_DATA_DIR)
```

Although I see an issue now when setting the user-data-dir while using ``uc`` mode, which leads to an odd URL at startup.

**Comment by M-Zubair10** [comment]
Is it possible to open 100 seleniumbase chromes all with different ips, I
attached 100 public ips to my virtual machine

I know this is offtopic, but I am new in python era and any help will be
really appreciated

Thanks

**Comment by mdmintz** [comment]
@M-Zubair10 Create a new discussion in https://github.com/seleniumbase/SeleniumBase/discussions for anything that isn't directly related to setting the ``user_data_dir`` when using ``uc`` mode. (When the PR I create for this original ticket gets merged, this ticket will close automatically.)

**Comment by mdmintz** [comment]
Released in ``4.7.0`` - https://github.com/seleniumbase/SeleniumBase/releases/tag/v4.7.0
